apps/openmw/mwrender/animation.cpp

```diff
@@ -1833,14 +1823,10 @@ namespace MWRender
         }
         else
         {
-            // TODO: use global attenuation settings
-
             // 1 pt of Light magnitude corresponds to 1 foot of radius
             float radius = effect * std::ceil(Constants::UnitsPerFoot);
-            const float linearValue = 3.f; // Currently hardcoded: unmodified Morrowind attenuation settings
-            float linearAttenuation = linearValue / radius;
 
-            if (!mGlowLight || linearAttenuation != mGlowLight->getLight(0)->getLinearAttenuation())
+            if (!mGlowLight || (radius * 3) != mGlowLight->getRadius())
```

Comment:
I'd prefer to keep the `linearValue` with comment. "3" is just a magic number here.

Reply:
It's not linearValue, it's the arbitrary multiplier chosen to reduce light pop-in as described below.

Reply:
Then create the variable with the `radius * 3` value, comment it and use it in both places.
Otherwise persons which will read this code will need to guess if both `3` are the same constant, or really different ones which accidentally got the same value in the current codebase.